pkg/apis/monitoring/v1alpha1/alertmanager_config_types.go

```diff
@@ -571,6 +573,16 @@ type HTTPConfig struct {
 	FollowRedirects *bool `json:"followRedirects,omitempty"`
 }
 
+// WebexConfig configures notification via Cisco Webex
+// See https://prometheus.io/docs/alerting/latest/configuration/#webex_config
+type WebexConfig struct {
+	SendResolved *bool       `json:"sendResolved,omitempty"`
+	APIURL       string      `json:"apiURL,omitempty"`
+	RoomID       string      `json:"roomID,omitempty"`
```

Comment:
room_id is a mandatory field.

```suggestion
	// ID of the Webex Teams room where to send the messages.
	// +kubebuilder:validation:MinLength=1
	RoomID       string      `json:"roomID,omitempty"`
```